build.gradle

```diff
@@ -34,26 +34,26 @@ ext {
     androidTargetSdkVersion = 34 // Android 14
     androidCompileSdkVersion = 34
```

Comment:
I am getting this error, migrate to 35

```
 Dependency 'androidx.core:core:1.15.0' requires libraries and applications that
           depend on it to compile against version 35 or later of the
           Android APIs.
```

After change build app to ensure no more errors exist.